MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/Attachment/AttachmentView.swift

```diff
@@ -46,12 +46,16 @@ public struct AttachmentView: View {
                                     }
                                 }()
                                 Spacer()
-                                TextField(placeholder, text: $viewModel.caption)
+                                TextField("", text: $viewModel.caption)
                                     .lineLimit(1)
                                     .textFieldStyle(.plain)
-                                    .foregroundColor(.white)
-                                    .placeholder(placeholder, when: viewModel.caption.isEmpty)
-                                    .padding(8)
+                                    .foregroundColor(.white).padding(EdgeInsets(top: 0, leading: 8, bottom: 0, trailing: 4))
+                                    .background(Color.black.opacity(0.4))
+                                    .placeholder(when: viewModel.caption.isEmpty) {
+                                        Text(placeholder).foregroundColor(.white).padding(EdgeInsets(top: 0, leading: 8, bottom: 0, trailing: 4))
+                                            .lineLimit(1)
+                                    }
+                                    .padding(EdgeInsets(top: 6, leading: 0, bottom: 10, trailing: 0))
```

Comment:
First of all: Thank you for your work 🥳 

When I run this, it's slightly better than before, but still difficult to see:

<img width="300px" src="https://user-images.githubusercontent.com/2580019/209565733-f494149f-f496-4394-9088-e9d9b7ee0e3a.png">

Do you know, what I mean? What do you think about this instead?

<img width="300px" src="https://user-images.githubusercontent.com/2580019/209565743-e5dccb94-3ecc-460b-a5f1-0f737a310ce0.png">

The color-values match (well, at least, they should) with what @j-f1 built for alt-text.

```swift
TextField("", text: $viewModel.caption)
    .lineLimit(1)
    .textFieldStyle(.plain)
    .padding(8)
    .placeholder(when: viewModel.caption.isEmpty) {
        Text(placeholder)
            .padding(8)
            .lineLimit(1)
    }
    .background(Color.black.opacity(0.85))
    .foregroundColor(.white)
    .padding(8)
```

One could throw in some rounded corners, too, but I'd leave that up to you :)

Reply:
Kind of obsolete now, as there should be a gradient?